File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        seen = [False] * 26
        
        for ch in sentence:
            seen[ord(ch) - ord('a')] = True
        
        for x in seen:
            if x == False:
                return False
        
        return True

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<bool> seen(26, false);
        
        for(char& ch: sentence) {
            seen[ch - 'a'] = true;
        }
        
        for(int i = 0; i < seen.size(); ++i) {
            if(seen[i] == false)
                return false;
        }
        return true;
    }
};